libsupport/src/EntityTypeManager.cpp

```diff
@@ -24,15 +24,9 @@ katana::EntityTypeManager::DoAssignEntityTypeIDsFromProperties(
   for (int i = 0, n = schema->num_fields(); i < n; i++) {
     const std::shared_ptr<arrow::Field>& current_field = schema->field(i);
 
-    // a bool or uint8 property is (always) considered a type
+    // a uint8 property is (always) considered a type
     // TODO(roshan) make this customizable by the user
-    if (current_field->type()->Equals(arrow::boolean())) {
-      type_field_indices.push_back(i);
-      std::shared_ptr<arrow::Array> property = properties->column(i)->chunk(0);
-      auto bool_property =
-          std::static_pointer_cast<arrow::BooleanArray>(property);
-      type_properties.bool_properties.emplace_back(i, bool_property);
-    } else if (current_field->type()->Equals(arrow::uint8())) {
+    if (current_field->type()->Equals(arrow::uint8())) {
       type_field_indices.push_back(i);
       std::shared_ptr<arrow::Array> property = properties->column(i)->chunk(0);
```

Comment:
I'm probably overly paranoid about this issue, but I might assert (debug?) that there is only one chunk.

Reply:
Will do